gnoi/os/client.go

```diff
@@ -62,10 +62,13 @@ func (c *Client) Activate(ctx context.Context, version string) error {
 
 // Verify invokes the Verify RPC for the OS service.
 func (c *Client) Verify(ctx context.Context) (version, activationFailMsg string, err error) {
+	request := &pb.VerifyRequest{}
+	utils.LogProto(request)
 	var out *pb.VerifyResponse
-	if out, err = c.client.Verify(ctx, &pb.VerifyRequest{}); err != nil {
+	if out, err = c.client.Verify(ctx, request); err != nil {
```

Comment:
pass &pb.VerifyRequest{} directly, no var assignment needed IMO.

Reply:
I'm not sure how I'll pass the request into `LogProto` when not assigning it

Reply:
would the following not work?

c.client.Verify(ctx, &pb.VerifyRequest{})

Reply:
sorry i missed the logging part :P